### PR TITLE
Navigation

### PR DESCRIPTION
**Pull request by jacwright**
Adds basic navigation and the action menu for creating folders/files (otherwise you can’t really see the navigation in action)

### CODE REVIEW

File: src/app/nav-item/template.html [hunk 1]
```diff
@@ -0,0 +1,54 @@
+<header [.open]="menu.open">
+  <div [if]="item.class !== 'Project' && item.children" (click)="toggleOpen()" class="toggle-arrow icon icon-folder-toggle"></div>
```
Comment (matthopson) [inline src/app/nav-item/template.html]: `&&` :grinning: so happy for these.

Comment (jacwright) [inline src/app/nav-item/template.html]: We don't support the browsers that didn't support those. :smile:

File: src/app/project/template.html [hunk 2]
```diff
@@ -1,9 +1,26 @@
...
+      <button (click)="window.history.forward()"><i class="icon icon-arrow-right"></i></button>
+    </div>
+  </div>
+
+  <dabble-navigation [unless]="loading" {project}="project"></dabble-navigation>
 </nav>
 
 <div id="editor">
-  <div id="project">
+  <div id="project" [if]="!loading">
```
Comment (matthopson) [inline src/app/project/template.html]: Just a question: Do you have any specific cases in which you'd use `[if]="!something"` over `[unless]="something"`

Comment (jacwright) [inline src/app/project/template.html]: No, `unless` is simply a more readable version. Which I forgot about using here.

**Comment by matthopson** [comment]
:+1:

**Pull request merged**